task_interpreter_ipf_client/models/client.py

```diff
@@ -140,18 +140,53 @@ def get_tolksprak(self):
         return self.get_request('/tolkportalen-tolkbokning/v1/tolksprak')
 
     @api.model
-    def populate_res_intepreter_language(self):
-        result = self.get_tolksprak()
+    def populate_all_data(self, silent=False):
+        if not (self.is_params_set() or silent):
+            raise Warning('All parameters are not set in config')
+        ok = True
+        for method, name, fields in (
+                (self.get_tolksprak, 'res.interpreter.language',
+                 (('name', 'namn'), ('code', 'id'))),
+                (self.get_kon, 'res.interpreter.gender_preference',
+                 (('name', 'namn'), ('code', 'id')))
+                ):
+
+            if not self.populate_data(method, name, fields):
+                msg = f'Failed to populate data for {name}'
+                _logger.warn(msg)
+                if not silent:
+                    raise Warning(msg)
+                ok = False
+        return ok
+
+    @api.model
+    def populate_data(self, method, name, fields):
```

Comment:
Seems that this model will only use this function. Can it be renamed to `_populate_data`? 
Also the parameter `name` can be changed to `language_name` for better clarity?